pump-jack-motion-monitoring/configuration/pump-jack-notecard-configuration.json

```diff
@@ -0,0 +1,55 @@
+# Tracking Device setup script version no. is published in an env var so
+# that it's visible within the Notehub's Device Environment Variable UI
+{"req":"env.default","name":"pump_jack_notecard_configuration_version","text":"1.0.0"}
+
+# Configure the Notecard's connection to Notehub.
+# Add a "product" key and value below to configure
+# to your project.
+{
+  "req":       "hub.set",
+  "mode":      "periodic",
+  "align":     true,
+  "voutbound": "usb:10; high:360; normal:720; low:1440; dead:0",
```

Comment:
Same comment as above - can't think why outbound would ever be configured to be bigger than inbound.